crates/matrix-sdk/src/widget/widget_settings/mod.rs

```diff
@@ -0,0 +1,393 @@
+pub use element_call::VirtualElementCallWidgetOptions;
+use language_tags::LanguageTag;
+use ruma::{api::client::profile::get_profile, DeviceId, RoomId, UserId};
+use url::Url;
+
+use crate::Room;
+
+mod element_call;
+
+mod url_params {
```

Comment:
Can you move this module to its own file please? Also, please move tests for `replace_properties` into the module then (in an inline `mod tests` like in this file).